doc/user/content/sql/create-sink.md

```diff
@@ -148,171 +129,19 @@ Note that:
 
 ### Kafka sinks
 
-When creating sinks, Materialize will either reuse the last sink topic (if `reuse_topic` is `true`) or it will generate a new topic name using the format below.
-```nofmt
-{topic_prefix}-{sink_global_id}-{materialize-startup-time}-{nonce}
-```
+When sinking into Kafka,
+Materialize will write all the changes from the specified source, table, or materialized view into the topic you specify.
```

Comment:
> from the specified source, table, or materialized view

This is where we'd want to introduce the term "durable relation", right? All good for now, just taking mental notes!

Reply:
Yes, I think so! (Not super attached to any specific term, but if we had one I'd definitely want to use it here.)